infra/main.parameters.json

```
@@ -98,6 +98,9 @@
     "chatGptDeploymentVersion":{
       "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_VERSION}"
     },
+    "chatGptDeploymentSkuName":{
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_SKUNAME}"
```

For similarity with other SKU variables in the JSON

```suggestion
      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_SKU}"
```